user_docs/en/userGuide.t2t

```diff
@@ -1934,6 +1934,19 @@ This slider allows you to set the volume of NVDA sounds and beeps.
 This setting only takes effect when "Volume of NVDA sounds follows voice volume" is disabled.
 This option is not available if you have started NVDA with [WASAPI disabled for audio output #WASAPI] in Advanced Settings.
 
+==== Duration of keeping audio device awake ====[KeepAudioAwakeDuration]
+
+This edit box specifies how long NVDA keeps the audio device awake after speech end.
+This allow to avoid certain glitches like dropped parts of speech sequences.
+This can happen due to audio devices (especially Bluetooth and wireless devices) entering stand-by mode.
+This might also be helpful in certain other use cases, such as running NVDA inside a virtual machine, such as Citrix Virtual Desktop, or on certain models of laptops.
+
+Lower values may mean audio would be cut-off more often, as a device may enter stand-by mode, causing the next speech to be clipped.
+A too high value may mean the battery of the sound output device discharges faster, as it stays active for longer while no sound is being sent.
+
+You can specify the duration to be 0 in order to disable this feature.

```

Comment:
This suggestion is applicable only if you change the word from "duration" to "time" as I commented above.
Note, that in networking and other compute contexts in my experience, the phrase is usually "keep-alive time", not "keep-alive duration". That may be why I mislike the word "duration" here.

```suggestion
You can set the time to zero in order to disable this feature.
```